services/src/test/scala/cromwell/pipeline/service/AuthServiceTest.scala

```diff
@@ -37,9 +36,6 @@ class AuthServiceTest extends WordSpec with Matchers with MockFactory {
 
   import authConfig._
 
-  private val userRepository: UserRepository = stub[UserRepository]
-  private val authUtils: AuthUtils = stub[AuthUtils]
-  private val authService: AuthService = new AuthService(userRepository, authUtils)
```

Comment:
Sharing mock between different tests leads to a lot of pain and bugs. I've moved them to `AuthServiceTestContext` below to ensure that each test has it's own mock.